src/autopas/tuning/AutoTuner.cpp

```diff
@@ -146,23 +149,23 @@ bool AutoTuner::tuneConfiguration() {
 
   // CASE: End of a tuning phase. This is not exclusive to the other cases!
   if (_configQueue.empty()) {
-    // if the queue is empty we are done tuning.
-    _iterationsSinceTuning = 0;
+    //  if the queue is empty we are done tuning.
+    _endOfTuningPhase = true;
     const auto [optConf, optEvidence] = _evidenceCollection.getOptimalConfiguration(_tuningPhase);
     _configQueue.push_back(optConf);
-    stillTuning = false;
+    _isTuning = false;
     // fill up sample buffer to indicate we are not collecting samples anymore
     _samplesRebuildingNeighborLists.resize(_maxSamples);
   }
   tuningTimer.stop();
 
-  return stillTuning;
+  return _isTuning;
 }
 
 const Configuration &AutoTuner::getCurrentConfig() const { return _configQueue.back(); }
 
 std::tuple<Configuration, bool> AutoTuner::getNextConfig() {
-  // If we are not (yet) tuning or there is nothing to tune return immediately.
+  //  If we are not (yet) tuning or there is nothing to tune return immediately.
```

Comment:
```suggestion
  // If we are not (yet) tuning or there is nothing to tune, return immediately.
```
Are the double spaces intentional?